pypsa/linopf.py

```diff
@@ -257,6 +258,41 @@ def define_unit_commitment_status_variables(n, sns, c):
     define_binaries(n, (sns, com_i), c, "status", mask=active)
 
 
+def define_modular_nominal(n, c, attr, attr_nom):
```

Comment:
@enricogiglio , we need to add a test to actually test this functionality.
Currently, the feature is not tested